filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -162,6 +162,10 @@ DECL_DRIVER_API_0(finish)
 // reset state tracking, if the driver does any state tracking (e.g. GL)
 DECL_DRIVER_API_0(resetState)
 
+DECL_DRIVER_API_N(setDebugTag,
+        backend::HandleBase::HandleId, handleId,
+        std::string&&, tag)
```

Comment:
we shouldn't use the STL here, ideally. Can we use `utils::CString` instead?

Reply:
also, I *think* you could use a by-value parameter here. This will leave the choice to the caller.
i.e. the caller can do std::move() if they wish and the cost will be identical. On the other hand, if they don't want to move the string, they also can.

Reply:
Yeah, let me see if `utils::CString` will work.

> the caller can do std::move() if they wish and the cost will be identical

This may be pedantic but a by-value parameter means the compiler still has to invoke the string's move constructor (though it's probably cheap), whereas passing by rvalue reference does not. Probably a wash in terms of performance though.